autodoc.py

```diff
@@ -0,0 +1,21 @@
+from docstring_parser import parse
```

Comment:
please fix lint issues. Also can we move this to `doc_utils/` where we will keep other docs helpers?